src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+
+const myProfile: React.FC = () => (
+  <div className="container w-96 mx-auto px-4">
+    <div style={
+      {
+        display: 'flex', justifyContent: 'flex-end'
+      }
+    }
+    >
+      <IconButton aria-label="Close" color="inherit">
+        <CloseIcon />
+      </IconButton>
+    </div>
+
+    <h1 className="font-bold mb-5 text-2xl font-inter">
+      My Profile
+    </h1>
+
+    <div className="w-full mb-4">
+      <label htmlFor="full-name" className="mb-1 text-xl font-medium font-inter">
+        Full Name
+      </label>
+      <br />
+      <input
+        type="text"
+        id="full-name"
+        className="w-full px-4 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500"
+        placeholder="John Doe"
+        required
+      />
+    </div>
+
+    <div className="w-full mb-4">
+      <label htmlFor="email" className="mt-1 mb-1 text-xl font-medium font-inter">
+        Email
+      </label>
+      <input
+        type="email"
+        id="email"
+        className="w-full px-4 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500"
+        placeholder="john.doe@website.com"
+        required
+      />
+    </div>
+
```

Comment:
Move label and input into separate component?

Reply:
add MyForm as a new component